source/configuration_manual/authentication/oauth2.rst

```diff
@@ -144,6 +144,48 @@ put into ``dovecot-oauth2.plain.conf.ext``
   username_attribute = username
   pass_attrs = host=127.0.0.1 proxy=y proxy_mech=xoauth2 pass=%{oauth2:access_token}
 
+Local validation
+****************
+
+.. versionadded:: 2.3.11
+
+Local validation allows validating tokens without connecting to an oauth2 server.
+This requires that key issuer supports `JWT tokens <https://jwt.io>`_.
+
+You can put the validation keys into any `dictionary <https://wiki.dovecot.org/Dictionary>`.
+The lookup key used is ``/shared/keyid``. If there is no ``kid`` element in token, ``default`` is used.
+Keys are cached into memory when they are fetched, and currently you need to restart dovecot to evict them from cache.
```

Comment:
The second part of this sentence reads a bit bumpy for me. 
Maybe like this?: `.. ,emory when they are fetched, to evict them from the cache you need to restart Dovecot.`